### DISCUSSION TITLE
7490 - encrypt backup

### DISCUSSION THREAD

**Discussion by flosch-dev** [Freetz-NG/freetz-ng#298] [Allgemein]
Hallo, 
kann man die Option "encrypt" backup zuverlässig benutzen oder kann es da noch Probleme geben? 
Welche kombination wäre zu empfehlen? 
- nur freetz (encrypt) + avm via AVM webif (encrypt)
- freetz + avm via freetz-web (encrypt)

Danke im Vorraus!

**Comment by fda77** [comment]
Die AVM Exports sind immer encrypted (die $$$$), entweder mit deiner mac+serial+x, oder mit Passwort. Ohne Passwort kann man diese dann nur genau auf dieser 1 Box restoren. Ich benutzte das nur wenn ich noch keine Freetz installiert hab.
Man konnte zwar schon immer mit Freetz AVM Backups machen, auf einer baugleichen Box mit andere Serial konnten die Passwörter aber nicht mehr von AVM gelesen werden. Daher nutze ich nur noch encrypted, falls die Box mal kaputt geht. Durch das enthaltene env können die Daten dann entschlüsselt werden.
Seit den letzten Änderungen wird auch geprüft ob es die gleiche Box ist und ein Restore überhaupt erfolgen sollte

**Comment by flosch-dev** [comment]
Danke für die Erklärung, dann lade ich mir beides encrypted via freetz webUI. - leider mit angezeigtem PW :)
